examples/NSW_community_gaming/tests/test_nsw_social_housie.catala_en

```diff
@@ -33,18 +33,18 @@ Message: unexpected token
 Autosuggestion: did you mean "content", or maybe "condition"?
 
 Error token:
-  --> examples/NSW_community_gaming/tests/test_nsw_social_housie.catala_en
```

Comment:
Can you also stylise this arrow ?

Reply:
I had trouble finding a similar unicode long arrow (or a hline that is always aligned with the normal → arrow). But I'll copy what ariadne does ^^